### DISCUSSION TITLE
UE 5.1 Support

### DISCUSSION THREAD

**Discussion by Jtheme** [Sixze/ALS-Refactored#178] [Q&A] [answered]
Has anyone been able to run this on 5.1 yet?

**Comment by Alex-G** [comment, marked as answer]
Jep,
I just compiled it, you get a message that it is made for UE 5.0 not UE 5.1.
You can edit `ALS.uplugin` to get rid of this message.
`"EngineVersion": "5.0.0"` -> `"EngineVersion": "5.1.0"`
![image](https://user-images.githubusercontent.com/5215249/202850020-5d83ee69-60fe-45f1-b5a4-8ca63643383a.png)
